tests/appsec/iast/source/test_header_name.py

```diff
@@ -17,6 +17,10 @@ class TestHeaderName(BaseSourceTest):
     source_type = "http.request.header.name"
     source_value = None
 
+    @bug(library="nodejs", reason="not reported correctly")
```

Comment:
As cookie name, is a bug or a nodejs limitation? if we can't implement it correctly, we should put it as `irrelevant` in nodejs.yml file instead of a bug here.